app/screens/edit_profile/edit_profile.tsx

```diff
@@ -0,0 +1,432 @@
+            isDisabled: (isSAMLOrLDAP(service) && lockedLastName) || includesSsoService(service),
+        },
+        username: {
+            ref: usernameRef,
+            isDisabled: service !== '',
+        },
+        email: {
+            ref: emailRef,
+            isDisabled: true,
+        },
+        nickname: {
+            ref: nicknameRef,
+            isDisabled: isSAMLOrLDAP(service) && lockedNickname,
+        },
+        position: {
+            ref: positionRef,
+            isDisabled: isSAMLOrLDAP(service) && lockedPosition,
+        },
+    };
+
+    const onFocusNextField = useCallback(((fieldKey: string) => {
+        const findNextField = () => {
+            const fields = Object.keys(userProfileFields);
+            const curIndex = fields.indexOf(fieldKey);
+            const searchIndex = curIndex + 1;
+
+            if (curIndex === -1 || searchIndex > fields.length) {
+                return undefined;
+            }
+
+            const remainingFields = fields.slice(searchIndex);
+
+            const nextFieldIndex = remainingFields.findIndex((f: string) => {
+                const field = userProfileFields[f];
+                return !field.isDisabled;
+            });
+
+            if (nextFieldIndex === -1) {
+                return {isLastEnabledField: true, nextField: undefined};
+            }
+
+            const fieldName = remainingFields[nextFieldIndex];
+
+            return {isLastEnabledField: false, nextField: userProfileFields[fieldName]};
+        };
+
+        const next = findNextField();
+        if (next?.isLastEnabledField && canSave) {
+            // performs form submission
+            Keyboard.dismiss();
+            submitUser();
+        } else if (next?.nextField) {
+            next?.nextField?.ref?.current?.focus();
+        } else {
+            Keyboard.dismiss();
+        }
+    }), [canSave]);
+
+    const fieldConfig: FieldConfig = {
+        blurOnSubmit: false,
+        enablesReturnKeyAutomatically: true,
+        onFocusNextField,
+        onTextChange: updateField,
+        returnKeyType: 'next',
+    };
+
+    return (
+        <>
+            {isTablet &&
+                <TabletTitle
+                    action={buttonText}
+                    enabled={canSave}
+                    onPress={submitUser}
+                    testID='custom_status.done.button'
+                    title={intl.formatMessage({id: 'mobile.screen.your_profile', defaultMessage: 'Your Profile'})}
+                />
+            }
+            <SafeAreaView
+                edges={edges}
+                style={styles.flex}
+                testID='edit_profile.screen'
+            >
+                <KeyboardAwareScrollView
+                    bounces={false}
+                    enableAutomaticScroll={true}
+                    enableOnAndroid={true}
+                    enableResetScrollToCoords={true}
+                    extraScrollHeight={25}
+                    extraScrollHeight={45}
```

Comment:
@avinashlng1080 this is duplicate, did you mean to have a different value per platform ?

Reply:
Thanks for catching that.   It got left out in the commits.  I intended to use only 45 for both platform